### PR TITLE
fix(api): export missing shadow dom strategies

### LINKED ISSUE #2688: bundle does not contain NativeShadowDomStrategy

### PR DESCRIPTION
**Pull request by 0x-r4bbit**
Can be merged once decided in #2688

Closes #2688

[Review on Reviewable](https://reviewable.io/reviews/angular/angular/2717)

### CODE REVIEW

**Comment by mhevery** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/angular/angular/2717)

File: modules/angular2/angular2.ts [hunk 1]
```diff
@@ -75,3 +75,5 @@ export * from './http';
 export {Observable, EventEmitter} from 'angular2/src/facade/async';
 export * from 'angular2/src/render/api';
 export {DomRenderer, DOCUMENT_TOKEN} from 'angular2/src/render/dom/dom_renderer';
+export {EmulatedScopedShadowDomStrategy} from 'angular2/src/render/dom/shadow_dom/emulated_scoped_shadow_dom_strategy';
```
Comment (tbosch) [inline modules/angular2/angular2.ts]: Let's reexport form `angular2/render` here, and use all 3 strategies.

**Comment by PascalPrecht** [comment]
@tbosch Updated the PR accordingly. Please not that I'm explicitly only exporting the Shadow DOM Strategies and didn't make any further changes, due to https://github.com/angular/angular/issues/2688#issuecomment-115567883

Let me know if I should change anything else!

**Comment by PascalPrecht** [comment]
<sup>**[modules/angular2/angular2.ts, line 78 \[r1\]](https://reviewable.io:443/reviews/angular/angular/2717#-JsgWWkentLx2owCxcR1-r1-78)** ([raw file](https://github.com/angular/angular/blob/7e5b11caccdcefe1a96c2e34c97d81d4f56f2d5e/modules/angular2/angular2.ts#L78)):</sup>
Done

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/angular/angular/2717)

**Comment by tbosch** [comment]
So there is a problem: If we export `ShadowDomStrategy`, we also need to export the classes `DomView`, `LightDom`, ...

However, all of this will change next week with https://github.com/angular/angular/issues/2529. I think it is actually not worth fixing Travis until then. Closing this PR...

**Pull request closed without merging**